scripts/globals/mobskills/Fluid_Toss_Claret.lua

```diff
@@ -0,0 +1,41 @@
+---------------------------------------------
+--  Fluid Toss
+--
+--  Description: Lobs a ball of liquid at a single target.
+--  Type: Ranged
+--  Utsusemi/Blink absorb: 1 shadow
+--  Range: Unknown range
+---------------------------------------------
+
+require("scripts/globals/settings");
+require("scripts/globals/status");
+require("scripts/globals/monstertpmoves");
+
+---------------------------------------------
+
+function onMobSkillCheck(target,mob,skill)
+	return 0;
+end;
+
+function onMobWeaponSkill(target, mob, skill)
+
+	local numhits = 1;
+	local accmod = 1;
+	local dmgmod = 5.5;
+	local info = MobPhysicalMove(mob,target,skill,numhits,accmod,dmgmod,TP_DMG_VARIES,1,2,3);
+	local dmg = MobFinalAdjustments(info.dmg,mob,skill,target,MOBSKILL_PHYSICAL,MOBPARAM_SLASH,info.hitslanded);
+	target:delHP(dmg);
+    
+    -- Apply poison
+    if (target:hasStatusEffect(EFFECT_POISON) == false) then
+        target:addStatusEffect(EFFECT_POISON, 100, 3, math.random(3,6) * 3); -- Poison for 3-6 ticks.
+    else
+        if (target:hasStatusEffect(EFFECT_POISON) == true) then
+            if (target:getStatusEffect(EFFECT_POISON):getPower() < 100) then
+                target:addStatusEffect(EFFECT_POISON, 100, 3, math.random(3,6) * 3); -- Poison for 3-6 ticks.                  

```

Comment:
I just checked and poison doesn't overwrite itself. You sure this is retail? Mobs won't use skills this quick so by the time the next fluit toss hits the target probably won't have it. I say just do `MobPhysicalStatusEffectMove(mob, target, skill, effect, power, tick, duration)`.

You also need to check for resistances and shadow hits, the method above dose all that for you.